src/routes/authentication.js

```diff
@@ -76,17 +84,14 @@ module.exports = function(app) {
           return shopModel.saveShop(shop, token);
         }).then((saveParam) => {
           winston.info('saved to db ' + saveParam);
-          return res.redirect(`${process.env.BASE_URL}/app_installed`);
-        }).catch((err) => res.redirect(`${process.env.BASE_URL}/app_installation_failed`));
+          const shop = req.query.shop;
+          const token = getJWTToken(shop);
+          res.redirect(`https://ja-admin-panel-us.herokuapp.com/?shop=${shop}&token=${token}`);
```

Comment:
I think it would be better to put this as environment variable. Maybe process.env.ADMIN_PANEL_URL